samples/cxx/custom/custom.cpp

```diff
@@ -1,7 +1,11 @@
 /*!
  * @file custom.cpp
- * Solve a closed-system constant pressure ignition problem where the governing equations
- * are custom-implemented.
+ *
+ * Custom Reactor
+ *
+ * Solve a closed-system constant pressure ignition problem where the governing
+ * equations are custom-implemented, using Cantera's interface to CVODES to
+ * integrate the equations.
```

Comment:
"Cantera's interface to CVODES to integrate" -> "Cantera's interface CVODES to integrate"

Reply:
Thanks @jongyoonbae! I think this is correct as written. Cantera provides an interfaces that connects to CVODES, which is actually does the numerical integration.